prompting/utils/config.py

```diff
@@ -286,7 +286,7 @@ def add_validator_args(cls, parser):
         type=float,
         nargs="+",
         help="The probability of sampling each task.",
-        default=[.25, .25, .25, 0, .25],
+        default=[.2, .2, .2, 0, .2, .2],
```

Comment:
This should be based on the task registry.
```suggestion
        from prompting.tasks import TASKS
        default=[1/len(TASKS) for task in TASKS],
```